emergence_lib/src/graphics/mod.rs

```diff
@@ -49,6 +49,8 @@ pub enum GraphicsStage {
     TilemapInitialization,
     /// Stage in which debug label generation happens, should run after [`TilemapInitialization`](GraphicsStage::TilemapInitialization)
     DebugLabelGeneration,
+    /// Stage in which GUi is drawn
+    GUIInitialization,
```

Comment:
super minor nit: the `GUIINitialization` stage is a little hard to read because of the unfortunate overlap between the `I` in `GUI` and the `I` in initialization,

I wonder if there are other terms

Reply:
GuiInitialization IMO

Reply:
GuiInitialization is in the newest commit